app/src/main/java/org/techtown/sopt_30th/seminar1_including_hw/HomeActivity.kt

```diff
@@ -0,0 +1,38 @@
+package org.techtown.sopt_30th.seminar1_including_hw
+
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import org.techtown.sopt_30th.R
+import org.techtown.sopt_30th.databinding.ActivityHomeBinding
+
+class HomeActivity : AppCompatActivity() {
+    private lateinit var binding: ActivityHomeBinding
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ActivityHomeBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+
+        initTransactionEvent()
+    }
+
+    private fun initTransactionEvent() {
+        val fragment1 = FollowerFragment()
+        val fragment2 = RepositoryFragment()
+
+        supportFragmentManager.beginTransaction().add(R.id.fragment_home, fragment1).commit()
```

Comment:
```suggestion
        supportFragmentManager.beginTransaction()
            .add(R.id.fragment_home, fragment1)
            .commit()
```
이렇게 작성하면 더 가독성이 좋을 것 같습니다. 윈도우에서 이렇게 자동으로 코드를 정렬해주는 단축키는 `CTRL+ALT+L`입니다.

Reply:
> 이렇게 작성하면 더 가독성이 좋을 것 같습니다. 윈도우에서 이렇게 자동으로 코드를 정렬해주는 단축키는 `CTRL+ALT+L`입니다.

ctrl alt L + ctrl alt O의 연속..감사합니다!